README.md

```diff
@@ -34,7 +34,7 @@ If your module is using the new node core [N-API][n-api], then you can prebuild
 prebuildify --napi
 ```
 
-Note that the flags `--all` and `--target` can add additional targets to the build, which likely are incompatible with N-API. Start with only the `--napi`flag, it should be sufficient for most scenarios.
+Note that the flags `--all` and `--target` can add additional targets to the build, which likely are incompatible with N-API. Start with only the `--napi` flag, it should be sufficient for most scenarios.
```

Comment:
Added two more sentences to make it even clearer that these flags may mess up the build for most users.

Reply:
> likely are incompatible with N-API

Oh, that's a good point. Could be considered a (small) bug on `--all`.

There is a good use case for `--target` though: building against an older version of node. This ensures that the included node headers are compatible with the minimum node version you support, as N-API promises forward compatibility, not necessarily backward compatibility.

I prefer that we instead add the caveat about `--napi --all` to the table of options. I'll make a suggestion.